PsfRuntime/CreateProcessHook.cpp

```diff
@@ -210,11 +210,20 @@ BOOL WINAPI CreateProcessFixup(
     fixupPath(finalPackagePath);
     fixupPath(exePath);
 
+    auto appConfig = PSFQueryCurrentAppLaunchConfig(true);
+    bool createProcessInAppContext = false;
+    auto createProcessInAppContextPtr = appConfig->try_get("InPackageContext");
+    if (createProcessInAppContextPtr)
+    {
+        createProcessInAppContext = createProcessInAppContextPtr->as_boolean().get();
+    }
+
 #if _DEBUG
     Log("\tPossible injection to process %ls %d.\n", exePath.data(), processInformation->dwProcessId);
 #endif
     if (((exePath.length() >= packagePath.length()) && (exePath.substr(0, packagePath.length()) == packagePath)) ||
-        ((exePath.length() >= finalPackagePath.length()) && (exePath.substr(0, finalPackagePath.length()) == finalPackagePath)))
+        ((exePath.length() >= finalPackagePath.length()) && (exePath.substr(0, finalPackagePath.length()) == finalPackagePath)) ||
+        (createProcessInAppContext))

```

Comment:
Can we add comment here adding condition to inject runtime dll to enable all further created process to run in package context ?